blog/2020-10-22-debezium-1-4-alpha1-released.adoc

```diff
@@ -0,0 +1,96 @@
+= Debezium 1.4.0.Alpha1 Released
+ccranfor
+:awestruct-tags: [ releases, mysql, postgres, mongodb, sqlserver, cassandra, db2, oracle, vitess ]
+:awestruct-layout: blog-post
+
+I am excited to announce the release of Debezium *1.4.0.Alpha1*!
+
+This first pass of the 1.4 release line provides a few useful new features:
+
+* New Vitess connector
+* Allow re-snapshot of specific tables
+
+Overall, the community fixed https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.4.0.Alpha1%20ORDER%20BY%20issuetype%20DESC[41 issues] for this release.
+Let's take a closer look at some of the highlights.
+
+== Vitess Connector
+
+https://www.vitess.io[Vitess] is a database solution for deploying, scaling, and managing large clusters of MySQL.
+We are excited that the Vitess engineers decided to build a CDC solution based on Debezium and to https://www.github.com/debezium/debezium-connector-vitess[open-source it] under the Debezium umbrella.
```

Comment:
Actually it's engineers of Bolt not Vitess itself who did this :) I've updated it.